assets/app/view/game/autoroute.rb

```diff
@@ -0,0 +1,14 @@
+module View
+  module Game
+    class Autoroute
+      def self.calculate(game, corporationId)
+        corp = game.corporation_by_id(corporationId)
+        trains = corp.trains
+
+        if trains.empty?
+          return []
```

Comment:
in ruby, the last line is always return, so you don't need to add a return

Reply:
I added `return` here because I'm going to add more after it.